### PR TITLE
K8s 1.15

### PR DESCRIPTION
**Pull request by tuommaki**
Towards https://github.com/giantswarm/giantswarm/issues/6302

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -74,21 +74,21 @@ required = [
   name = "github.com/prometheus/client_golang"
   version = "0.9.3"
 
-[[constraint]]
+[[override]]
```
Comment (tuommaki) [inline Gopkg.toml]: This is rude, but seems to be the only way to work out the circle with e2e-libraries.

**Comment by xh3b4sd** [comment]
I pushed some changes. This is now left. 

```
$ o test ./...
# github.com/giantswarm/helmclient/vendor/k8s.io/apiextensions-apiserver/pkg/features
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:35:27: undefined: feature.Feature
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:42:29: undefined: feature.Feature
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:48:34: undefined: feature.Feature
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:52:32: feature.DefaultFeatureGate.Add undefined (type featuregate.FeatureGate has no field or method Add)
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:58:41: undefined: feature.Feature
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:58:61: undefined: feature.FeatureSpec
FAIL	github.com/giantswarm/helmclient [build failed]
FAIL	github.com/giantswarm/helmclient/helmclienttest [build failed]
```

**Comment by rossf7** [comment]
@tuommaki Taking over here Thx!

**Comment by sslavic** [comment]
Closing in favor of https://github.com/giantswarm/helmclient/pull/119

**Pull request closed without merging**